packages/xo-web/src/xo-app/settings/remotes/remote.js

```diff
@@ -381,7 +409,6 @@ export default decorate([
                   onChange={effects.setSecretKey}
                   placeholder='Paste secret here to change it'
                   autoComplete='off'
-                  required
```

Comment:
Is this change related to this PR?

Reply:
The bug was found when developing this code.